docker/ghd

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Link me into your $PATH as `ln -s ../path/to/ghd`
+
+script_path() {
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+        READLINK="readlink"
+    else
+        READLINK="readlink -f"
+    fi
+
+    echo "$(dirname "$($READLINK "${BASH_SOURCE[0]}")")"
+}
+
+current_dir="$(pwd -P)"
+
+pushd "$(script_path)" > /dev/null
+
+    set +e
+
+    poetry run bash -c " cd \"$current_dir\" ; ghd $@ "
```

Comment:
```suggestion
    poetry run bash -c " cd \"$current_dir\" && ghd \"$@\""
```

I'm not sure if this will cause infinite recursion in certain cases.

Reply:
`;` -> `&&` is fine, but I think it would be only relevant if some race occurs, which would cause the directory to be deleted before the command is executed - and this would mean we have a much worse problem somewhere else :)

Quotes around `$@` are wrong I think - did you actually try it? I get the following:

```
zaytsev@work:~$ ghd
Usage: ghd [OPTIONS] COMMAND [ARGS]...
Try "ghd --help" for help.

Error: No such command "".
```